src/test_l1_dsDisplay.c

```diff
@@ -731,7 +731,7 @@ void test_l1_dsDisplay_negative_dsGetEDIDBytes(void) {
     UT_LOG("\n In %s Return value: [%d]\n", __FUNCTION__, result);
     if(result != dsERR_NOT_INITIALIZED)
     {
-        UT_FAIL("Incorrect error return");
+        UT_FAIL_FATAL("Incorrect error return");
```

Comment:
whether UT_FAIL_FATAL will exit here from the test? If yes whether it will proceed to free the memory? Also, if yes we may need to see UT_FAIL_FATAL used below in the function that may exit without free the memory.